onnxscript/ir/passes/common/unused_removal_test.py

```diff
@@ -13,15 +13,13 @@
 class RemoveUnusedTest(unittest.TestCase):
     using_ir: bool
 
-    def remove_unused_nodes(
-        self, model: onnx.ModelProto, remove_initialized_inputs: bool = False
-    ):
+    def remove_unused_nodes(self, model: onnx.ModelProto):
         if self.using_ir:
             model_ir = ir.serde.deserialize_model(model)
-            onnxscript.optimizer.remove_unused_nodes(model_ir, remove_initialized_inputs)
+            onnxscript.optimizer.remove_unused_nodes(model_ir)
             model = ir.serde.serialize_model(model_ir)
             return model
-        onnxscript.optimizer.remove_unused_nodes(model, remove_initialized_inputs)
+        onnxscript.optimizer.remove_unused_nodes(model)
         return model
 
     def test_remove_unused_nodes(self):
```

Comment:
There should a test with a model including subgraphs.

Reply:
Thanks - will update

Reply:
https://github.com/microsoft/onnxscript/issues/2264